src/cell.rs

```diff
@@ -41,8 +41,8 @@ impl Cell {
 
     /// Create a new `Cell` initialized with content from `string`.
     /// By default, content is align to `LEFT`
-    pub fn new(string: &str) -> Cell {
-        Cell::new_align(string, Alignment::LEFT)
+    pub fn new<S: ToString>(string: S) -> Cell {
```

Comment:
Although perhaps you don't want that, dunno, up to you, you're the crate author ! :)